### ISSUE TITLE
Add overrides for liblwjgl.so 

### ISSUE THREAD

**Issue by Richard-Rogalski** [ATLauncher/ATLauncher#754]
### The Problem

No easy way to import custom overrides for libraries and such

### The Solution

Hey!! Multimc has this cool option where you can add overrides like this: https://forums.raptorcs.com/index.php/topic,118.0.html

Would this be possible in ATLauncher? Most I see is system openal & glfw. I'm on PPC64le, so I need to use a patched liblwjgl too. I'm going to try to hack something together for the time being (to prove that atlauncher on ppc64le would otherwise work) but it'll be pretty unwieldly and prone to being overwritten.

Another thing that would be really appreciated is only enable things that use native libs, like discord presence libs, upstream JDK, openal, glfw, etc, if on x86/64(or arm, if you've made it that far yet :). 

Another idea is instead of adding an easier override system, maybe hosted bins of these liblwjgl's? although upstream liblwjgl doesn't support linux/x86 or ppc, it's simple enough to get it working on these targets, and it can autodetect from the arch and use these libs (like the example from that link above).

### Have you checked this issue doesn't already exist?

- [X] I am sure this issue doesn't exist already

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by RyanTheAllmighty** [comment]
It's entirely possible that we could provide advanced library management as part of #556. I'd keep this issue here separate though, as I probably wouldn't include it by default as part of that as it's quite an advanced thing and something that definitely needs more thought from our side as to how we allow it.

I'm all for it though, definitely a niche thing I would think, but can be done

**Comment by Richard-Rogalski** [comment]
glad to hear there's hope :)

I tried to override the .jar with the native libs, but every time I try launching the game it overrides it again. Will try to spend some time on this tomorrow :)

**Comment by RyanTheAllmighty** [comment]
Yeah the game will self heal any missing/corrupt libraries, so just replacing it will cause it to be redownloaded.

You can open up the `instance.json` file in the instances root, find the library you want to correct and change the url, hash and size of the download in there and it should work how you want for now
